index.js

```diff
@@ -59,6 +59,23 @@ module.exports = {
     // Prefer named export
     'import/prefer-default-export': 'off',
     'import/no-default-export': 'error',
+
+    // Order import with line separation between groups
+    'import/order': [
+      'error',
+      {
+        // Add module import (@Module) after external import
```

Comment:
Do not relate to business logic (`module`) here, module is only sensfull in some projects.